src/main/java/com/powsybl/openloadflow/network/impl/LfBusImpl.java

```diff
@@ -400,20 +399,16 @@ public void updateState(boolean reactiveLimits) {
         updateGeneratorsState(voltageControl ? calculatedQ + loadTargetQ : generationTargetQ, reactiveLimits);
 
         // update load power
+        double dp = positiveLoadCount > 0 ? (loadTargetP - initialLoadTargetP) / positiveLoadCount : 0;
```

Comment:
Hi Geoffroy, thank you for the fix ! But the algorithm here is quite different that the first one: if you have more than one load in a bus, the slack has an equal distribution between them. If you have a big one and a very small one, it could lead to problems. I am not sure that the ````positiveLoadCount```` is needed. Do you see ? You can a positive load in a bus in the unit test.